doc/man/usbguard-rules.conf.5.adoc

```diff
@@ -292,7 +292,7 @@ The following set of rules forms a policy that allows USB flash disks and explic
 ....
 +
 The policy rejects all USB flash disk devices with an interface from the HID/Keyboard, Communications and Wireless classes.
-Note that blacklisting is the wrong approach and you shouldn't just blacklist a set of devices and allow the rest.
+Note that default allow is the wrong approach and you shouldn't just reject a set of devices and allow the rest.
```

Comment:
Shouldn't this be `default deny` here?

@radosroka

Reply:
Also @eiais

Reply:
> Shouldn't this be `default deny` here?
> 
> @radosroka

No, it is correctly stated that default allow is the wrong approach